src/scripts/offscreen.js

```diff
@@ -1,7 +1,5 @@
 var a = 0, b = 0
 
-const dataMigrationService = new DataMigrationService()
```

Comment:
this is the issue , i suspect `openDatabase` has already ben removed even with the origin trails, this line fails with 

```websql-database.js:5 Uncaught ReferenceError: openDatabase is not defined
    at new WebSQLDBService (websql-database.js:5:5)
    at new DataMigrationService (data-migration.js:4:24)
    at offscreen.js:3:30
```

and everything else below it breaks. That being said , i wonder if we should just remove the migration logic at this point but users risk not getting jobs migrated